tests/csm/rest/test_capacity_quota.py

```diff
@@ -641,3 +646,354 @@ def test_40640(self):
+
+    @pytest.mark.skip("Feature not ready")
+    @pytest.mark.lc
+    @pytest.mark.csmrest
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.parallel
+    @pytest.mark.tags('TEST-41156')
+    def test_41156(self):
+        """
+        Test GET capacity stats for create IAM user and Put objects using manage users
+        """
+        test_case_name = cortxlogging.get_frame()
+        self.log.info("##### Test started -  %s #####", test_case_name)
+        test_cfg = self.csm_conf["test_41156"]
+        max_size = test_cfg["max_size"]
+        random_size = self.cryptogen.randrange(1, max_size)
+        num_objects = math.floor(max_size / random_size)
+        self.log.info("Step 1: Creating IAM users with different tenant")
+        for cnt in range(2):
+            tenant = "tenant_" + str(cnt)
+            self.log.info("Creating new iam user with tenant %s", tenant)
+            optional_payload = self.csm_obj.iam_user_payload_rgw("loaded")
+            optional_payload.update({"tenant": tenant})
+            self.log.info("updated payload :  %s", optional_payload)
+            resp1 = self.csm_obj.create_iam_user_rgw(optional_payload)
+            self.log.info("Verify Response : %s", resp1)
+            assert_utils.assert_true(resp1.status_code == HTTPStatus.CREATED,
+                                     "IAM user creation failed")
+            self.created_iam_users.add(resp1.json()['tenant'] + "$" + optional_payload['uid'])
+            resp = self.csm_obj.compare_iam_payload_response(resp1, optional_payload)
+            self.log.info("Printing response %s", resp)
+            assert_utils.assert_true(resp[0], resp[1])
+            self.log.info("Step 2: Verify Create bucket: %s with access key: %s and secret key: %s",
+                          self.bucket, self.akey, self.skey)
+            bucket_created = s3_misc.create_bucket(self.bucket, self.akey, self.skey)
+            assert bucket_created, "Failed to create bucket"
+            self.log.info("Step 3: Create N objects of total size S")
+            for _ in range(0, num_objects):
+                resp = s3_misc.create_put_objects(self.obj_name, self.bucket,
+                                                  self.akey, self.skey, object_size=random_size)
+                assert resp, "Put object Failed"
+            self.log.info("Step 4: Perform GET API to get capacity usage")
+            resp = self.csm_obj.get_capacity_usage("user", self.user_id)
+            assert resp.status_code == HTTPStatus.BAD_REQUEST, \
+                "Status code check failed for get capacity"
+            self.log.info("Step 5: Verify above count matches aws response")
+            status, output = run_local_cmd(commands.CMD_AWSCLI_READABLE, chk_stderr=True)
+            assert status, "list objects operation failed"
+            total_objects, total_size = self.csm_obj.get_objects_and_size(output)
+            t_obj = resp.json()["capacity"]["s3"]["user"][0]["objects"]
```

Comment:
U need to calculate the object size written and verify against that also
So AWS response == CSM response == what was written

Reply:
added those verifications